tools/check/ut.go

```diff
@@ -341,14 +355,15 @@ type numa struct {
 	numactl bool
 }
 
-func (n *numa) worker(wg *sync.WaitGroup, ch chan task) {
+func (n *numa) worker(wg *sync.WaitGroup, ch chan task, ErrCh chan struct{}) {
 	defer wg.Done()
 	for t := range ch {
 		start := time.Now()
 		res := n.runTestCase(t.pkg, t.test, t.old)
 		if res.err != nil {
 			fmt.Println("[FAIL] ", t.pkg, t.test, t.old, time.Since(start), res.err)
 			io.Copy(os.Stderr, &res.output)
+			ErrCh <- struct{}{}
```

Comment:
This code is buggy, if there are several error send to the ErrCh, the code will block here forever

Reply:
Removed